src/api/flash/FocusCommands.js

```diff
@@ -30,6 +30,14 @@ export function FocusCommands(options) {
     let port = focus._port;
 
     if (devicePort && focusDeviceDescriptor) {
+      if (port?.isOpen) {
```

Comment:
Since we repeat this pattern multiple times, maybe we should move it into focus?